src/lib/auth/authconf.rb

```diff
@@ -755,18 +758,70 @@ def aux_apply
                 service_disable_stop('nscd')
             end
         end
-        
+
+        def is_installed_version_newer_or_equal?(installed_rpm_version, test_rpm_version)
+            installed_rpm_version_l = installed_rpm_version
+                .split(/[-.+]/)
+                .select { |i| i.match?(/^\d+$/) }
+                .map(&:to_i)
+
+            test_rpm_version_l = test_rpm_version
+                .split(/[-.+]/)
+                .select { |i| i.match?(/^\d+$/) }
+                .map(&:to_i)
+
+            Yast::Builtins.y2milestone(
+                "Evaluating installed %1 and test %2 versions",
+                installed_rpm_version_l,
+                test_rpm_version_l
+            )
```

Comment:
Sorry, I overlooked this in the first review. I would simply use `log.info` intead of `Builtins`:

```suggestion
            log.info(
                "Evaluating installed #{installed_rpm_version_l} and test #{test_rpm_version_l} versions"
            )
```